docs/sdk-and-tools/sdk-py/installing-mxpy.md

```diff
@@ -3,15 +3,15 @@ id: installing-mxpy
 title: Installing mxpy
 ---
 
-[comment]: # (mx-abstract)
+[comment]: # "mx-abstract"
```

Comment:
revert these changes - they are needed by the bot that splits the docs into granular sections

Reply:
Reverted